### PR TITLE
Add ERC-20 tokens FOX and WIS to all_assets

### PR DESCRIPTION
**Pull request by davidhildebrand**
This pull request contains the addition of two ERC-20 tokens (FOX, WIS) to collection in `all_assets.json`.

### CODE REVIEW

File: rotkehlchen/data/all_assets.json [hunk 1]
```diff
@@ -5085,6 +5085,15 @@
         "symbol": "FOTA",
         "type": "ethereum token"
     },
+    "FOX": {
+        "coingecko": "fox-token",
+        "ethereum_address": "0xc770EEfAd204B5180dF6a14Ee197D99d808ee52d",
+        "ethereum_token_decimals": 18,
+        "name": "FOX Token",
+        "started": 1592535600,
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:5093]: Shouldn't this be `1553639047`?
Seems to have been deployed much earlier: https://etherscan.io/tx/0x590c4ad4f2ea141cffa2dcf10230afc48778c85ce785bcfb96a12fb509dd26ce

File: rotkehlchen/data/all_assets.json [hunk 2]
```diff
@@ -12754,6 +12763,15 @@
         "symbol": "WINGS",
         "type": "ethereum token"
     },
+    "WIS": {
+        "coingecko": "experty-wisdom-token",
+        "ethereum_address": "0xDecade1c6Bf2cD9fb89aFad73e4a519C867adcF5",
+        "ethereum_token_decimals": 18,
+        "name": "Experty Wisdom Token",
+        "started": 1605558693,
```
Comment (LefterisJP) [inline rotkehlchen/data/all_assets.json:12771]: This also seems a little bit off: 1604695924 is the deployment timestamp of the ethereum contract: https://etherscan.io/tx/0x1fb141389e9c4e2529b99cb4c04d22283253ad9b38773c498f20c756906282f7

**Review by LefterisJP — COMMENTED** [review]
Hey thanks a lot for the PR!

Looks good. Left you 2 comments for the timestamps to fix. Also can you:

1. Sign the CLA so your commits can be accepted?
2. Add a changelog entry. Changelog is [here](https://raw.githubusercontent.com/rotki/rotki/develop/docs/changelog.rst) and the entry can look like:
```
* :feature:`-` Added support for the following tokens:

  - `Experty Wisdom Token (WIS) <https://www.coingecko.com/en/coins/experty-wisdom-token>`__
   ...
   ....
   etc.
```
Should be placed right above the last released version.
3. Can you also adjust the expected md5 and version in the assets test? https://github.com/rotki/rotki/blob/8d75f951ab36fa68fcf51c26b02c8e874ef79574/rotkehlchen/tests/unit/test_assets.py#L286

Thanks a lot!

File: rotkehlchen/data/all_assets.json (hunk 1 above)
Comment (davidhildebrand) [reply rotkehlchen/data/all_assets.json:5093]: Yes!  I was working off the earliest datetime seen on CoinGecko, but I see now how to go about this properly.  Thanks!

File: rotkehlchen/data/all_assets.json (hunk 2 above)
Comment (davidhildebrand) [reply rotkehlchen/data/all_assets.json:12771]: Got it.  Thanks!

**Review by LefterisJP — APPROVED** [review]
Thanks a lot @davidhildebrand. Looks good!

The failing e2e test is unrelated and something flaky we need to look at.

**Pull request merged**